train_segmentation_model.py

```diff
@@ -82,7 +82,8 @@ def train(gcp_bucket, config_file):
              'validation').as_posix(),
         rescale=1./255,
         target_size=target_size,
-        batch_size=batch_size)
+        batch_size=batch_size,
+        seed=train_config['validation_data_shuffle_seed'])
```

Comment:
if I'm reading this right -- I think if that key is not set in the config this line will crash the program. do we want that? or do we want to check to see if it's given, and if not, then just don't use a seed?

Reply:
```python
None if 'validation_data_shuffle_seed' not in train_config else train_config['validation_data_shuffle_seed']
```

Reply:
(same for train seed)

Reply:
done for train, val, and test